app/Repositories/NeedsRepository.php

```diff
@@ -83,7 +83,7 @@ public function searchNeeds($search){
                     $query = $query->where(DB::Raw('CONCAT_WS(" ",name,telephone,address,city,needs)'),"like","%$searchPart%");
```

Comment:
@nadeeravista could you please have a look on @gayanhewa's comment as well as fix conflicts?